spec/globus_client/endpoint_spec.rb

```diff
@@ -256,23 +256,7 @@
   end
 
   describe "#disallow_writes" do
-    let(:fake_identity) { instance_double(GlobusClient::Identity, get_identity_id: "example") }
```

Comment:
user_id is `nil` when calling `disallow_writes` from the consumer, so an identity should not be mocked in tests.